lguohan/sonic-buildimage
# Added LPM and reset support to sfputil for MLNX msn2700 platform [NOT FOR master]

**For review ONLY**
**PR can be merged after integrating MLNX SDK version 4.2.7000X042**
**Also the support for other MLNX platform will be added very soon**
**Currently support is available only for SN2700** 

**- What I did**
Added LPM and reset support to sfputil for MLNX platforms 

**-How I did it**
Implemented additional scripts which uses a MLNX SDK

**- How to verify it**
1. For reset - check the admin status during restart
2. For LPM:
2.1 Get LPM status
2.2 Set LPM to OFF
2.3 Get LPM status and check if it's OFF
2.4 Set LPM to ON
2.5 Get LPM status and check if it's ON

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
